clouddriver-google/clouddriver-google.gradle

```diff
@@ -47,8 +47,15 @@ dependencies {
   testImplementation "org.springframework:spring-test"
   testImplementation "org.springframework.boot:spring-boot-test"
   testImplementation "org.codehaus.groovy:groovy-test"
+  testImplementation 'org.spockframework:spock-core:2.0-groovy-3.0'
+  testImplementation 'org.powermock:powermock-api-mockito2:2.0.9'
+  testImplementation 'org.powermock:powermock-module-junit4:2.0.9'
 }
 
 configurations.all {
-   resolutionStrategy.force 'com.google.apis:google-api-services-compute:beta-rev20201102-1.30.10'
+   resolutionStrategy {
```

Comment:
This doesn't work the way most think it does.  Particularly with the kork PR, these resolution strategy blocks need to be removed.

Reply:
I can get rid of those but we need to merge and release https://github.com/spinnaker/kork/pull/1203 first to ensure the correct version is installed.